internal/internal_worker.go

```diff
@@ -58,6 +58,8 @@ const (
 
 	defaultMaxConcurrentWorkflowExecutionSize = 50     // hardcoded max workflow execution size.
 	defaultMaxWorkflowExecutionRate           = 100000 // Large workflow execution rate (unlimited)
+
+	testTagsContextKey = "testTags"
```

Comment:
to avoid key conflict, we should use a more explicit key name, like "cadence_testTags".

Reply:
ok